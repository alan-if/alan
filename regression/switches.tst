../compiler/alan -h
../interpreter/arun -n -h
../interpreter/arun -n -help
