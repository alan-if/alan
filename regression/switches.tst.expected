########## switches ##########
../compiler/alan -h
../interpreter/arun -n -h
../interpreter/arun -n -help
########## /bin/sh  switches.tst ##########
Parameter error: Ambiguous option, -h
Usage: ALAN <adventure> [-help] [options]
Unrecognized switch, -h
Usage:

    arun [<switches>] <adventure>

where the possible optional switches are:
    -v    verbose mode
    -l    log player commands and game output to a file
    -i    ignore version and checksum errors
    -n    no Status Line
    -d    enter debug mode
    -t    trace game execution
    -s    single instruction trace

Unrecognized switch, -h
Usage:

    arun [<switches>] <adventure>

where the possible optional switches are:
    -v    verbose mode
    -l    log player commands and game output to a file
    -i    ignore version and checksum errors
    -n    no Status Line
    -d    enter debug mode
    -t    trace game execution
    -s    single instruction trace

